code.gs

```diff
@@ -1387,20 +1408,32 @@ function main (forceDate) {
       .concat(isRunningOutdatedVersion() ? ['<br/><br/><b>', htmlEscape(bodySuffixes[2]), ' <a href="', baseGitHubProjectURL, 'releases/latest', '">', htmlEscape(bodySuffixes[3]), '</a>.</b></p>'] : ['</p>'])
       .join('');
 
-    // ...send the email notification.
-    log.add('Sending email...', 'info');
-    MailApp.sendEmail({
-      to: settings.user.notificationEmail,
-      subject: subject,
-      body: body,
-      htmlBody: htmlBody,
-      inlineImages: inlineImages,
-      name: settings.user.emailSenderName
-    });
-    log.add('Email sent.', 'info');
+    // ...return mail content.
+
+    var emailData = {
+      'subject': subject, 
+      'body': body, 
+      'htmlBody': htmlBody, 
+      'inlineImages': inlineImages
+    }
+    
+    return emailData;
   }
-  // Send the log if the debug options say so.
-  log.sendEmail(settings.user.notificationEmail, settings.user.emailSenderName);
+}
+
+/*
+ * Send email with provided content
+ */
+function sendEmail (subject, body, htmlBody, inlineImages) {
```

Comment:
I know I am questioning my own suggestion here, but I don't think this needs to exist as separate function.  
We can call the `MailApp.sendEmail()` function (and log the event with `log.add()`) directly on line R1226 as well as in `unit-tests.gs` when it will be needed: declaring this function does not simplify anything.

Sorry for this messy back and forth.